### ISSUE TITLE
Flathub Proton-GE not detected

### ISSUE THREAD

**Issue by buxel** [Heroic-Games-Launcher/HeroicGamesLauncher#309]
**Describe the bug**
Flatpak version of Proton GE is not recognized. If this is not supported in general, please consider this a feature request :-)

**To Reproduce**
- no wine installed on my system
- install Proton GE according to https://github.com/GloriousEggroll/proton-ge-custom#flatpak
- try to find it in heroic
- no luck

I also tried adding the flatpak path to heroic:
`/var/lib/flatpak/runtime/com.valvesoftware.Steam.CompatibilityTool.Proton-GE/x86_64/stable/active/files`

...but this was expected to fail:
```
Launch Command:    STEAM_COMPAT_DATA_PATH='/home/chris/.wine'   /opt/heroic/resources/app.asar.unpacked/build/bin/legendary launch  Blowfish  --no-wine --wrapper "'/var/lib/flatpak/runtime/com.valvesoftware.Steam.CompatibilityTool.Proton-GE/x86_64/stable/active/files/proton' run"  
Wine not installed
Wine not installed
Wine not installed
launching Blowfish
GameMode not installed

```

**Expected behavior**
It is possible to use Proton GE somehow. Steam(also flatpak) picks it up without issues.

**Desktop (please complete the following information):**

- OS: OpenSuse
- Version Tumbleweed
- Game: FTL
- Wine/Proton version: Proton-GE (flatpak)

**Comment by adityaruplaha** [comment]
Yeah Flatpak version are not supported yet, we're looking into it in the future.

**Comment by flavioislima** [comment]
Unfortunatelly we will never support this because this version of Proton-GE is specific to the Steam Flatpak version, and it cant work with Heroic. 

It is written on their repository:

```
Flatpak
This section is for those that use the Steam flatpak.
```

For Heroic, download the standard version and extract it under ` .config/heroic/tools/proton/`

**Comment by flavioislima** [comment]
> @flavioislima Sorry to comment after years, but after Wine-GE being deprecated and UMU being integrated into Proton-GE, maybe this issue can be reopened.
> I gave to Heroic Launcher access to `/var/lib/flatpak/runtime/com.valvesoftware.Steam.CompatibilityTool.Proton-GE/x86_64/stable/active`, added `/var/lib/flatpak/runtime/com.valvesoftware.Steam.CompatibilityTool.Proton-GE/x86_64/stable/active/files/proton` as a custom Proton, and all the games I'm testing are working perfectly with it.
> It's very convenient, this runtime is shared with Steam, it's updated regularly, so it's kind of "set and forget".

OK, thanks for bringing this up. 
I reopened so we can investigate it in the future.

**Comment by arielj** [comment]
Closing this.

The flatpak GE-proton is [not official and not maintained but GE nor valve](https://github.com/GloriousEggroll/proton-ge-custom?tab=readme-ov-file#flathub), we should encourage users to use the version we download from GE's repository and not encourage the use of the flatpak.
